ultimatejobweb/templates/available_jobs.html

```diff
@@ -72,8 +72,17 @@ <h1 class="display-3">UltimateJob</h1>
                 <th>Link to job application</th>
             </tr>
         </thead>
-        <tbody>
-            <tr>
+        <tbody>-->
+		   <h1> Available jobs  </h1>
+		   <div class="showSearch">
+		   {% for item in job %}
+			<tr> 
+				<td>{{ item.JobTitle }}</td>
```

Comment:
You probably want to add:
The name of the hiring company
The URL of the job connected to the JobTitle.